stylesheets/layout_desktop.scss

```diff
@@ -45,4 +45,42 @@ section.curriculum .image-blurb {
 }
 .hero-blurb p {
   background-color: rgba(0, 0, 0, 0.03);
+}
+h3 {  font-size: modular-scale(0); }
+nav { ul { font-size: modular-scale(-1); } }
```

Comment:
There's no reason to nest if we're just including one additional thing in the brackets, so let's remove the ```{``` after nav.